tests/utils/fund/engine.ts

```diff
@@ -0,0 +1,55 @@
+import { BigNumberish, Signer } from 'ethers';
```

Comment:
This file should not be in the `fund` directory. The `fund` directory is for utilities around fund creation, etc. This one could just be in `utils/engine.ts` and exported from `utils/index.ts` too.